src/widgets/pixbuf_loader.rs

```diff
@@ -43,7 +43,7 @@ impl PixbufLoader {
     ///
     /// # Failures
     /// Returns `None` if the pixbuf loader cannot be created.
-    pub fn new_with_type(image_type: &str, error: &mut ::glib::Error) -> Option<PixbufLoader> {
+    pub fn new_with_type(image_type: &str, error: &mut Error) -> Option<PixbufLoader> {
```

Comment:
Sounds good to me.  Pursuant to your comment in https://github.com/rust-gnome/gdk/issues/4#issuecomment-98530710, I will update the functions to match this style:
```rust
should_not_return_NULL_but_might_because_ffi_craziness() -> T // with the possibility of a panic!
might_validly_return_NULL() -> Option<T>
might_return_GError() -> Result<T, Error>
```
In this particular case, these will panic if `tmp` is `NULL`.